openmc/executor.py

```diff
@@ -32,17 +34,56 @@ def plot_geometry(output=True, openmc_exec='openmc', cwd='.'):
 
     Parameters
     ----------
-    output : bool
+    output : bool, optional
         Capture OpenMC output from standard out
-    openmc_exec : str
+    openmc_exec : str, optional
         Path to OpenMC executable
     cwd : str, optional
-        Path to working directory to run in. Defaults to the current working directory.
+        Path to working directory to run in
 
     """
     return _run([openmc_exec, '-p'], output, cwd)
 
 
+def plot_inline(plots, openmc_exec='openmc', cwd='.', convert_exec='convert'):
+    """Display plots inline in a Jupyter notebook.
+
```

Comment:
Perhaps include an example of what `convert_exec` might be on a Mac. I don't think `convert` is installed (at least by default).